auth/auth.go

```diff
@@ -96,7 +97,14 @@ func HandleAuthCallback(w http.ResponseWriter, r *http.Request) {
 	functions[handle_callback].(func(http.ResponseWriter, *http.Request))(w, r)
 }
 
-// HandleAuthLogin - handles oauth login
```

Comment:
Is it possible to leave public function comments?

Reply:
you mean like this?  If so, seems to work..  I can put the Swagger comments before the regular function comments.

// swagger:route GET /api/oauth/login nodes HandleAuthLogin
//
// Handles OAuth login
//
//		Schemes: https
//
// 		Security:
//   		oauth
// HandleAuthLogin - handles oauth login